### PR TITLE
Exercise 4: HTML tables

### PR DESCRIPTION
**Pull request by karlysun**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +38,17 @@
         
         <section class="schedule">
             <h2>Schedule</h2>
+            <table>
+                <tr class="bold">
+                    <td>Morning</td>
+                    
+                    <td>Afternoon</td>
+                </tr>
```
Comment (nyanofthemoon) [inline index.html]: @karlysun Please read the documentation relating to `th` versus `td` applying to table headers.

**Comment by karlysun** [comment]
Updated

**Comment by karlysun** [comment]
Updated again to make table looks like the one on the readme (On github)

**Comment by nyanofthemoon** [comment]
Thank you! :+1:

**Pull request closed without merging**